src/Resolver/Computed.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Magento\Upward\Resolver;
+
+use Magento\Upward\Definition;
+use Magento\UpwardConnector\Model\Computed\ComputedInterface;
+
+class Computed extends AbstractResolver
+{
+    /** @var \Magento\UpwardConnector\Model\ComputedPool */
+    private $computed;
+
+    public function __construct()
+    {
+        $this->computed = \Magento\Framework\App\ObjectManager::getInstance()
```

Comment:
Should be possible to create this via dependency injection `ObjectManage` is only allowed for Factory and also not in constructor

Reply:
@larsroettig Unfortunately they instantiate the class directly with `new self::$resolverClasses[$resolverType]()` so dependency injection won't work. This is essentially a way to go from UPWARD (no dependency injection) back into Magento coding practices (with dependency injection).